### DISCUSSION TITLE
Unsupported device (ENVILAR ZG_LED_DRIVER42CC)

### DISCUSSION THREAD

**Discussion by 2proof4u** [Koenkk/zigbee2mqtt#13013] [Devices]
Hi

I have read the "support new device" and had luck creating a .js file that got it to work.

I wanted to add the pic and upload the device as described so others can use it but i am very unsure in understanding what I have to do. 
Is there some sort of youtube guide I can follow so i dont do anything wrong? (cant seam to find one by myself)

Info
It is a envilar ZG_LED_DRIVER42CC
Link to pic
https://www.plusled.dk/led-paneler/tilbehor-til-led-paneler/zigbee-zg42cc-uni-leddriver.html

**Comment by Koenkk** [comment]
Can you share your `.js` file? Then I can add it, the picture will be added with the next release.

**Reply by 2proof4u** [reply]
Hi

Here it is
[envilar.zip](https://github.com/Koenkk/zigbee2mqtt/files/9031701/envilar.zip)

**Reply by Koenkk** [reply]
Thanks ,added it so the device will be supported out-of-the-box in the next z2m release (1 Aug) including the picture.

**Reply by droneman83** [reply]
is there a way to reset the led driver, befor i have triede to set it up for Home assistant it was in Smartthings, i have triede many ways to reset it and do come with different ways of flashing 1 sek on and 1 sek off, pulsing slow, pulsing and 2 sek on and 2 sek off, but im not able to add it to z2m??

**Comment by 2proof4u** [comment]
Super

Thanks for your work
